docs/_templates/changelog/v3.0.0.md

```diff
@@ -0,0 +1,25 @@
+## 📦 [Bookshelf - 3.0.0](https://github.com/mcbookshelf/Bookshelf/releases/tag/v3.0.0)
+
+> **🚂 The Pipeline Update!**
+>
+> *This release overhauls the project structure with Beet, focusing on modularity and isolation of each module. It also introduces a new version loader that resolves conflicts between multiple versions.*
+>
+> **⚠️ BREAKING CHANGES**: This new version has a few breaking changes!
+
+- ⚠️ **[#281](https://github.com/mcbookshelf/Bookshelf/issues/281)** - Merged the `bs.biome` and `bs.weather` modules into a single `bs.environment` module.
+- 🎉 **[#272](https://github.com/mcbookshelf/Bookshelf/issues/272)** - Added a new automatically generated `bs.load` module, eliminating the need to manually create `load` and `unload` tags for each module. Bookshelf now also uses **LanternLoad** to load its modules.
```

Comment:
I'm not sure this description is enough the breaking change. Not having to manually create files is not a breaking change for me.
Moreover, it may be useful to add a link on "LanternLoad" to this project to better understand what is it.

Reply:
The breaking change is for the module merge (biome and weather) and since we are planning more BC i think it's fair to keep it here. +1 for the link

Reply:
Okay my bad, I didn't udnerstand that breaking changes are indicated through the warning emoji. Perhaps we can explicit that?

Reply:
Sure we can maybe add a header that explains this in the changelog. Also there is already a file that shows this inside docs/_templates/changelog/template.md

Reply:
Yeah but I suppose this file is more for us/contributors than for documentation users. I think it's better to have a "legend" section as header of each changelog page.